CONTRIBUTING.md

```diff
@@ -42,5 +42,31 @@ We recommend VS Code with ESLint, Flow, and Prettier (with prettier-eslint enabl
    yarn prettier
    ```
 4. **Mark your changes in CHANGELOG**
-   
+
    Put a one-line description of your change under Added/Changed section. See [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Editing native code
+
+In `native/ios` and `native/android` you'll find the native bridge code for React Native.
+
+It's recommended to use the latest stable version of Xcode / Android Studio to work on that code.
+
+### Integration tests
+
+If you change native bridge code or `adapter/sqlite` code, it's recommended to run integration tests that run the entire Watermelon code with SQLite and React Native in the loop:
+
+```bash
+yarn test:ios
+yarn test:android
+```
+
+You can also run iOS integration tests manually by opening the `native/iosTest/WatermelonTester.xcworkspace` project and hitting Cmd+U.
```

Comment:
Giving a recommendation on how to run it in XCode but not in Android Studio. Should I add line about Android?

Reply:
Yeah!